components/carousel/index.jsx

```diff
@@ -0,0 +1,287 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// # Carousel Component
+
+// Implements the [Carousel design pattern](https://www.lightningdesignsystem.com/components/carousel/) in React.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// ### shortid
+// [npmjs.com/package/shortid](https://www.npmjs.com/package/shortid)
+// shortid is a short, non-sequential, url-friendly, unique id generator
+import shortid from 'shortid';
+import '../../styles/carousel/carousel.css';
```

Comment:
We can't import CSS into these components. I don't think one of the builds will take it (I think it's the ES6 build). We can document this in the component comments for the doc site though.

Reply:
@interactivellama need to discuss for clarification